pyproject.toml

```diff
@@ -8,23 +8,19 @@ readme = "README.md"
 homepage = "https://beaker.algo.xyz"
 repository = "https://github.com/algorand-devrel/beaker"
 keywords = ["pyteal", "algorand"]
-classifiers = [
-  "Development Status :: 2 - Pre-Alpha",
-]
-packages = [
-    { include = "beaker" },
-]
+classifiers = ["Development Status :: 2 - Pre-Alpha"]
```

Comment:
If we're gonna modify this bit anyway, it should be:
`Development Status :: 5 - Production/Stable`